icons/slack.svg

```diff
@@ -4,14 +4,7 @@
0.5 13ZM15.5 13L15.5 12L15.5 12L15.5 13ZM20.5 16L20.5 17H20.5L20.5 16ZM15.5 16L15.5 15L15.5 15L15.5 16ZM14 14.5L13 14.5L14 14.5ZM22 14.5L21 14.5L22 14.5ZM14 20.5L13 20.5L13 20.5L14 20.5ZM14 19V18C13.4477 18 13 18.4477 13 19H14ZM15.5 19L15.5 20L15.5 20L15.5 19ZM17 20.5L18 20.5L17 20.5ZM8.5 11L8.5 12L8.5 12L8.5 11ZM3.5 11L3.5 10L3.5 10L3.5 11ZM8.5 8L8.5 9L8.5 9L8.5 8ZM3.5 8L3.5 7L3.5 7L3.5 8ZM2 9.5L3 9.5L2 9.5ZM10 3.5L11 3.5L11 3.5L10 3.5ZM10 5V6C10.5523 6 11 5.55228 11 5H10ZM8.5 5L8.5 4L8.5 4L8.5 5ZM8.5 2L8.5 1L8.5 2ZM7 3.5L6 3.5L7 3.5ZM14 8.5V3.5H12V8.5H14ZM15 3.5V8.5H17V3.5H15ZM15 8.5C15 8.77614 14.7761 9 14.5 9V11C15.8807 11 17 9.88071 17 8.5H15ZM14.5 3C14.7761 3 15 3.22386 15 3.5H17C17 2.11929 15.8807 1 14.5 1V3ZM12 8.5C12 9.88071 13.1193 11 14.5 11V9C14.2239 9 14 8.77614 14 8.5H12ZM14 3.5C14 3.22386 14.2239 3 14.5 3V1C13.1193 1 12 2.11929 12 3.5H14ZM20.5 9H19V11H20.5V9ZM20 10V8.5H18V10H20ZM21 8.5C21 8.77614 20.7761 9 20.5 9V11C21.8807 11 23 9.88071 23 8.5H21ZM20.5 8C20.7761 8 21 8.22386 21 8.5H23C23 7.11929 21.8807 6 20.5 6V8ZM20.5 6C19.1193 6 18 7.11929 18 8.5H20C20 8.22386 20.2239 8 20.5 8V6ZM10 15.5V20.5H12V15.5H10ZM7 15.5V20.5H9V15.5H7ZM7 20.5C7 21.8807 8.11929 23 9.5 23V21C9.22386 21 9 20.7761 9 20.5H7ZM9.5 13C8.11929 13 7 14.1193 7 15.5H9C9 15.2239 9.22386 15 9.5 15V13ZM12 15.5C12 14.1193 10.8807 13 9.5 13V15C9.77614 15 10 15.2239 10 15.5H12ZM10 20.5C10 20.7761 9.77614 21 9.5 21V23C10.8807 23 12 21.8807 12 20.5H10ZM3.5 15H5V13H3.5V15ZM4 14V15.5H6V14H4ZM3 15.5C3 15.2239 3.22386 15 3.5 15V13C2.11929 13 1 14.1193 1 15.5H3ZM3.5 16C3.22386 16 3 15.7761 3 15.5H1C1 16.8807 2.11929 18 3.5 18V16ZM3.5 18C4.88071 18 6 16.8807 6 15.5H4C4 15.7761 3.77614 16 3.5 16V18ZM20.5 12H15.5V14H20.5V12ZM20.5 15H15.5V17H20.5V15ZM15.5 15C15.2239 15 15 14.7761 15 14.5L13 14.5C13 15.8807 14.1193 17 15.5 17L15.5 15ZM21 14.5C21 14.7761 20.7761 15 20.5 15L20.5 17C21.8807 17 23 15.8807 23 14.5L21 14.5ZM15.5 12C14.1193 12 13 13.1193 13 14.5L15 14.5C15 14.2239 15.2239 14 15.5 14L15.5 12ZM20.5 14C20.7761 14 21 14.2239 21 14.5L23 14.5C23 13.1193 21.8807 12 20.5 12L20.5 14ZM15 20.5V19H13V20.5H15ZM14 20H15.5V18H14V20ZM15.5 21C15.2239 21 15 20.7761 15 20.5L13 20.5C13 21.8807 14.1193 23 15.5 23L15.5 21ZM16 20.5C16 20.7761 15.7761 21 15.5 21L15.5 23C16.8807 23 18 21.8807 18 20.5L16 20.5ZM18 20.5C18 19.1193 16.8807 18 15.5 18L15.5 20C15.7761 20 16 20.2239 16 20.5L18 20.5ZM8.5 10H3.5V12H8.5V10ZM8.5 7L3.5 7L3.5 9L8.5 9L8.5 7ZM3.5 7C2.11929 7 1 8.11929 1 9.5L3 9.5C3 9.22386 3.22386 9 3.5 9L3.5 7ZM11 9.5C11 8.11929 9.88071 7 8.5 7L8.5 9C8.77614 9 9 9.22386 9 9.5L11 9.5ZM8.5 12C9.88071 12 11 10.8807 11 9.5L9 9.5C9 9.77614 8.77614 10 8.5 10L8.5 12ZM3.5 10C3.22386 10 3 9.77614 3 9.5L1 9.5C1 10.8807 2.11929 12 3.5 12L3.5 10ZM9 3.5V5H11V3.5H9ZM10 4H8.5V6H10V4ZM8.5 3C8.77614 3 9 3.22386 9 3.5L11 3.5C11 2.11929 9.88071 1 8.5 1L8.5 3ZM8 3.5C8 3.22386 8.22386 3 8.5 3L8.5 1C7.11929 1 6 2.11929 6 3.5L8 3.5ZM6 3.5C6 4.88071 7.11929 6 8.5 6L8.5 4C8.22386 4 8 3.77614 8 3.5L6 3.5Z" fill="black"/>
```

Comment:
Looks like you may have converted stroke to outlines here.

Reply:
Can you export a stroke version instead?

Reply:
Also, don't forget to run `npm run build` after adding a new svg 😄

Reply:
weird, i didn't flatten it but i'll go ahead and double check later.